lib-cloudsdk/src/main/java/com/facebook/openwifi/cloudsdk/models/ap/UCentralSchema.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.openwifi.cloudsdk.models.ap;
+
+import java.util.List;
+
+import com.google.gson.JsonPrimitive;
+import com.google.gson.annotations.SerializedName;
+
+public class UCentralSchema {
```

Comment:
Not sure whether it will be useful or not: do you think it might be helpful to put the schema link here so people can know what we are referring to quickly (so they don't need to be like us searching around all the time)? Thanks.

Reply:
Good idea, will follow up separately.